content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -248,15 +243,15 @@ sudo systemctl restart sensu-agent
 
 **2. Install Nginx and the Sensu HTTP Plugin**
 
-We'll use the [Sensu HTTP Plugin](https://github.com/sensu-plugins/sensu-plugins-http) to monitor an Nginx server running on the sandbox.
+You'll use the [Sensu HTTP Plugin](https://github.com/sensu-plugins/sensu-plugins-http) to monitor an Nginx server running on the sandbox.
 
 First, install and start Nginx:
 
 {{< highlight shell >}}
 sudo yum install -y nginx && sudo systemctl start nginx
```

Comment:
Unrelated to this PR, this command find nginx for me. It might be due to me having to use CentOS 7.4, but not sure. Should confirm with another user.

Reply:
Cameron: need to run before `yum install -y nginx`
`yum install -y epel-release`

Reply:
Confirmed installing EPEL resolved the issue.